pages/1.10/installing/installation-faq/index.md

```diff
@@ -50,20 +50,48 @@ If Auto Scaling groups are in use, the node will be replaced automatically.
     ```bash
     sudo systemctl kill -s SIGUSR1 dcos-mesos-slave-public && sudo systemctl stop dcos-mesos-slave-public
     ```
+
+[enterprise]
 ## Q. How do I backup the IAM database?
+[/enterprise]
 
-- To backup the IAM database to a file run the following command on one of the master nodes:
+To backup the IAM database to a file run the following command on one of the master nodes:
 
-```bash
-dcos-shell iam-database-backup > ~/iam-backup.sql
-```
+    ```bash
```

Comment:
It looks like the formatting is broken here.

Reply:
That was intentional. The scripts that are used to convert the markdown to html actually need those indents to format the code block properly.